app/assets/javascripts/discourse/app/components/post-text-selection-toolbar.gjs

```diff
@@ -196,16 +177,8 @@ export default class PostTextSelectionToolbar extends Component {
   }
 
   <template>
-    {{! template-lint-disable no-invalid-interactive }}
-    {{! template-lint-disable no-pointer-down-event-binding }}
     <div
-      {{on "mousedown" this.trapEvents}}
-      {{on "mouseup" this.trapEvents}}
```

Comment:
we are ok hiding the toolbar when losing selection now, so we don't need all these hacks anymore